app/views/directives/notifications/notification-body.html

```diff
@@ -71,7 +72,7 @@
         </span>
 
         <span ng-if="!(notification.event.involvedObject)">
-          {{notification.message}}
+          <span ng-bind-html="notification.message"></span>
```

Comment:
Can't assume that the message is HTML. We need a flag that says use HTML for the message. Otherwise plain text messages will not properly be escaped.

Reply:
Ok, should I check the message for HTML tags, or if I detect that a custom message is being used, is that enough to process it as HTML?

Reply:
No way to reliably detect if it has markup. I would document that the custom quota messages are markup. Then add another a flag like `notification.isHTML` that tells the drawer how to render it.